psd.h

```diff
@@ -257,6 +461,10 @@ namespace psd {
     // Writes one byte in the file (or do nothing if ok() = false)
     virtual void write8(uint8_t value) = 0;
     virtual bool write(const uint8_t* buf, uint32_t size) = 0;
+
+    virtual bool write16(const uint16_t w) = 0;
+    virtual bool write32(const uint32_t l) = 0;
+    virtual bool write64(const uint64_t l) = 0;
   };
```

Comment:
Don't add new member functions to `FileInterface`. write8 and write should be enough to implement all those functions (write16/32/64) later outside the `FileInterface` class (i.e. inside the `Encoder`, just like in the `Decoder`).